tools/js-tools/eslintrc/base.mjs

```diff
@@ -325,6 +325,18 @@ export default function makeBaseConfig( configurl, opts = {} ) {
 					'error',
 					{ allowInterfaces: 'with-single-extends' },
 				],
+				// Disable param name check for destructured params.
+				'jsdoc/check-param-names': [ 'error', { checkDestructured: false } ],
+				// TS should be enough to catch undefined types.
+				'jsdoc/no-undefined-types': 'warn',
+				// Disable param check except when destructured.
+				'jsdoc/require-param': [ 'error', { checkDestructured: false } ],
+				// TS should mostly have the type set.
+				'jsdoc/require-param-type': 'off',
+				'jsdoc/require-property-type': 'off',
+				// Let us use TS return type for better inference
+				'jsdoc/require-returns': 'off',
```

Comment:
Why are you disabling this? jsdoc's `recommended-typescript` config doesn't do this, it only turns off `jsdoc/require-returns-type`.

What this seems to disable is it warning about the entirely missing `@returns` in something like this:
```ts
/**
 * @param opts - Options
 */
function xxx( { foo: string, bar: number } ): null {
    return null;
}
```
Which doesn't seem to have anything to do with types.

Reply:
It got in there while debuggin. Restored ✅  

Thanks.